change/@griffel-eslint-plugin-5d860b0c-ef67-4015-a825-eae4b10d1636.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
Staging the rule should not affect public API - the rule and none of its utilties are exported